tests/e2e/workloads/amq/test_amq_pod_respin.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.framework.testlib import (
+    E2ETest, workloads, ignore_leftovers
+)
+from tests.helpers import default_storage_class
+from tests.disruption_helpers import Disruptions
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.pod import get_all_pods
+
+
+log = logging.getLogger(__name__)
+
+
+def respin_amq_app_pod(kafka_namespace):
+    """
+    Respin amq pod
+
+    Args:
+        kafka_namespace (str): Namespace for kafka
+
+    """
+    pod = ocp.OCP(kind=constants.POD, namespace=kafka_namespace)
+    pod_obj_list = get_all_pods(namespace=kafka_namespace)
+    for pod_obj in pod_obj_list:
+        pod_obj.delete()
+        assert pod.wait_for_resource(
+            condition='Running', resource_count=len(pod_obj_list), timeout=300
+        )
+
+
+@ignore_leftovers
```

Comment:
Any reason for ignoring leftovers?

Reply:
yes, Respin pod creates new pod so at the end of the test fails with leftovers